saleor/order/actions.py

```diff
@@ -120,7 +120,7 @@ def order_created(
         )
 
     channel = order_info.channel
-    if channel.automatically_confirm_all_new_orders or from_draft:
+    if channel.automatically_confirm_all_new_orders:
```

Comment:
Did you maybe thought about adding additional flag to cover that behavior of draft orders?
I think it makes more sense to not have, but just leaving a note.

Anyway we are missing the changelog for that change and it should land there I believe, and we should update graphql description for that change.